src/App/components/Sidebar/SidebarSearchResults/PoolsSearchResults/PoolsSearchResults.tsx

```diff
@@ -52,16 +57,52 @@ export default function PoolsSearchResults(props: propsIF) {
     };
 
     return (
-        <div>
-            <h4 className={styles.card_title}>Pools</h4>
+        <FlexContainer
+            flexDirection='column'
+            justifyContent='center'
+            alignItems='flex-start'
+            gap={8}
+        >
+            <Text fontWeight='500' fontSize='body' color='accent5'>
```

Comment:
Might also be nice to have constants for the different weights we use.  Or at least in the Text component have a subset of weights available. Just so I don't type in 450 and suddenly we're using slightly different weights throughout.

Reply:
will be done in Phase 2